src/utils/processLinkData.js

```diff
@@ -0,0 +1,18 @@
+import { format } from "date-fns";
+import processTimeData from "./processTimeData";
+
+//시간 형식 바꿔주기 위해!!
+const processLinkData = (link) => {
+  const { id, createdAt, url, imageSource, description } = link;
```

Comment:
P3:
다른 매개변수를 받지 않는다면 아래처럼도 가능합니다.

```suggestion
const processLinkData = ({ id, createdAt, url, imageSource, description }) => {
```